### PR TITLE
Exit popup modal (x)

### LINKED ISSUE #254: Exit (X) in a pop up Modal
### Description:

Add an exit  (X) in the `HOW DO I CHOOSE STATUS` pop up modal,  this should be mobile friendlier as well.

Can also fix typo on the info. `Whatever you `writ` here will be `visable` to staff and to the person you are reviewing. Take the time to give an accurate and useful review` , change visible to `visible` and `writ` to `write`

### Steps to reproduce:

1. Go to a specific card in the Review or Feedback Review Column,
2. Click on Add Review button
3. Click on the `HOW DO I CHOOSE  STATUS` button,

### Expected behaviour:

Users should be able to click on the exit (X) weather in mobile or Desktop to close the Pop up modal

Screenshot 2021-11-18 at 17 18 55

### Actual behaviour:

Screenshot 2021-11-18 at 17 10 02

Screenshot 2021-11-18 at 17 27 42

### Client Setup Information

- Browser Version:
- Operating System:

### PR DESCRIPTION
**Pull request by Vuyisanani**
Related issues: [please specify]

## Description:
When on a mobile device, we could not close the help popup , so we added an exit (x)

What are you up to? Fill us in :)

## Screenshots/videos

Screenshot 2021-12-02 at 07 05 01

Screenshot 2022-01-27 at 08 20 48

## I solemnly swear that:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by babalwa-mbolekwa — CHANGES_REQUESTED** [review]
Hey Viro. Looks good. could you please also style the (X) so that it shows that it is clickable. maybe it could be inside a circle and also a pointer when you hover around it? and could you please make it a lowercase x

**Review by babalwa-mbolekwa — CHANGES_REQUESTED** [review]
@Vuyisanani could you please also fix the functionality, when I click on the modal the second time it doesn't open, and shows the following instead:
modal

**Review by babalwa-mbolekwa — APPROVED** [review]

File: frontend/src/components/regions/AddCardReviewModal/index.js [hunk 1]
```diff
@@ -44,13 +44,18 @@ const AddReviewModalUnconnected = ({
     saveReview({ status, comments, cardId });
   };
 
+  const closeHelpPopUp = (e) => {
+    document.getElementById("popUpInfo").remove();
```
Comment (sheenarbw) [inline frontend/src/components/regions/AddCardReviewModal/index.js]: I think we should use React mechanisms here, don't interact directly with the dom.  Are we managing the opening and closing of this modal using state or redux?  Please use the mechanisms in place.

**Review by sheenarbw — CHANGES_REQUESTED** [review]

**Review by sheenarbw — CHANGES_REQUESTED** [review]
Why should only one Modal have an X. Why not all our modals?

**Review by babalwa-mbolekwa — APPROVED** [review]

File: frontend/src/components/regions/AddCardReviewModal/Presentation.js [hunk 2]
```diff
@@ -38,12 +38,32 @@ const useStyles = makeStyles((theme) => ({
...
+const StatusHelp = ({ closePopUpInfo, closeHelpPopUp }) => {
+const classes = useStyles();
+  
   return (
     <Help buttonText="How do I choose a status?">
-      <Paper>
+
+      <Paper ref={closePopUpInfo} className={classes.popUpInfo}>
+        <Typography variant="subtitle2" className={classes.helpPopUpInfo} onClick={closeHelpPopUp}>
+          X
```
Comment (Sbonelo01) [inline frontend/src/components/regions/AddCardReviewModal/Presentation.js:64]: Hi there @Vuyisanani , why not make use of the exit icon found in the material UI icons ?

**Review by Sbonelo01 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**